README.md

```diff
@@ -34,17 +34,16 @@ account, settlement and funding currency.
 - RON (Romanian leu)
 - CZK (Czech koruna)
 - BGN (Bulgarian lev)
-- HRK (Croatian kuna)
 
 ## Usage
 
 Within the Paylike ecosystem you should use the `code` to refer to a currency.
 
 ```js
-var currencies = require('@paylike/currencies');
+var currencies = require("@paylike/currencies");
```

Comment:
What's going on here? We usually don't use double quotes for strings.

Reply:
missing prettier config